src/commands/react-native/README.md

```diff
@@ -100,6 +100,37 @@ This allows Datadog to create proper URLs such as:
 | Bitbucket | https://\<repository-url\>/src/\<commit-hash\>/\<tracked-file-path\>#lines-\<line\>  |
 
 
+### `codepush`
+
+This command uploads your JavaScript source maps and their corresponding bundle file to Datadog after an AppCenter Codepush build.
```

Comment:
Is there just one bundle file for all of the source maps? If there are multiple bundle files, please make "files" plural.

Reply:
For codepush it's only one bundle file :)